packages/mobile/src/components/drawers/ConfirmationDrawerWithoutRedux.tsx

```diff
@@ -0,0 +1,43 @@
+import type { SetOptional } from 'type-fest'
+
+import Drawer from '../drawer'
+import type { DrawerProps } from '../drawer/Drawer'
+
+import type { BaseConfirmationDrawerProps } from './ConfirmationDrawer'
+import { ConfirmationDrawerContent } from './ConfirmationDrawer'
+
+type ConfirmationDrawerWithoutReduxProps = BaseConfirmationDrawerProps &
+  SetOptional<DrawerProps, 'children'>
+
+/** ConfirmationDrawer that isn't coupled to redux */
+export const ConfirmationDrawerWithoutRedux = (
```

Comment:
Could this just be part of the existing confirmation drawer file, since that has the web redux vs native redux stuff right?

Reply:
I wanted to keep it separate to make it obvious that the behavior is different, also following the same pattern with ActionDrawerWithoutRedux